Build-V3-Plugin-Monitoring-ios/Classes/TracesManagers/FirebasePerformanceTracesManager.swift

```diff
@@ -12,6 +12,9 @@ import WebKit
 
 public protocol FirebasePerformanceDelegate {
     func updateTraceAdditional(attributes: [String : String]) -> [String : String]
+    func performance_traceName(for event: SDKEvent) -> String?
```

Comment:
I think the naming is not clear, this method is called for the naming of SDK events

```Swift 
func performanceTraceNameForSDKEvent(for event: SDKEvent) -> String
```